django_project/core/base_templates/sidebar.html

```diff
@@ -8,6 +8,13 @@
 			{{ form_data.provider_name }}
 		</div>
 		<div class='col-md-12 noplr'>
+            {% if provider.name %}
+                <ul>
+                    <li>
+                        <a href="/ford3/providers/{{ provider.id }}">{{ provider.name }}</a>
```

Comment:
See line 8 of this same file `form_data.provider_name`.
No need for a list here, a `h5` or `h6` would be enough.